packages/blade/src/components/Dropdown/docs/DropdownWithSelect.stories.tsx

```diff
@@ -299,6 +299,29 @@ export const InternalSelect = (): React.ReactElement => {
   );
 };
 
+export const InternalDisabledSelect = (): React.ReactElement => {
+  const [isDisabled, setIsDisabled] = React.useState(false);
+  return (
+    <Box padding="spacing.5" maxWidth="400px">
+      <Button marginBottom="spacing.4" isFullWidth onClick={() => setIsDisabled(!isDisabled)}>
+        Toggle Input
```

Comment:
shouldn't the caption be Toggle Disabled State?

Reply:
Yup right. Resolved in 0eaae714b9264c398313a222891e18e4053750be